diff --git a/node_modules/cofhejs/dist/node.js b/node_modules/cofhejs/dist/node.js
index 59f7216..def9e3e 100644
--- a/node_modules/cofhejs/dist/node.js
+++ b/node_modules/cofhejs/dist/node.js
@@ -27467,7 +27467,7 @@ var MAX_UINT128 = 340282366920938463463374607431768211455n;
 var MAX_UINT256 = 115792089237316195423570985008687907853269984665640564039457584007913129640319n;
 var PUBLIC_KEY_LENGTH_MIN = 15e3;
 var TaskManagerAddress = "0xeA30c4B8b44078Bbf8a6ef5b9f1eC1626C7848D9";
-var MockZkVerifierAddress = "0x0000000000000000000000000000000000000100";
+var MockZkVerifierAddress = "0x0000000000000000000000000000000000000101";
 var MockQueryDecrypterAddress = "0x0000000000000000000000000000000000000200";
 var MockZkVerifierSignerPkey = "0x6c8d7f768a6bb4aafe85e8a2f5a9680355239c7e14646ed62b044e39de154512";
 var fnExistsSig = "function exists() public view returns (bool)";
diff --git a/node_modules/cofhejs/dist/node.mjs b/node_modules/cofhejs/dist/node.mjs
index abd9e06..eda0ea8 100644
--- a/node_modules/cofhejs/dist/node.mjs
+++ b/node_modules/cofhejs/dist/node.mjs
@@ -27441,7 +27441,7 @@ var MAX_UINT128 = 340282366920938463463374607431768211455n;
 var MAX_UINT256 = 115792089237316195423570985008687907853269984665640564039457584007913129640319n;
 var PUBLIC_KEY_LENGTH_MIN = 15e3;
 var TaskManagerAddress = "0xeA30c4B8b44078Bbf8a6ef5b9f1eC1626C7848D9";
-var MockZkVerifierAddress = "0x0000000000000000000000000000000000000100";
+var MockZkVerifierAddress = "0x0000000000000000000000000000000000000101";
 var MockQueryDecrypterAddress = "0x0000000000000000000000000000000000000200";
 var MockZkVerifierSignerPkey = "0x6c8d7f768a6bb4aafe85e8a2f5a9680355239c7e14646ed62b044e39de154512";
 var fnExistsSig = "function exists() public view returns (bool)";
